cms/djangoapps/contentstore/views/item.py

```diff
@@ -295,6 +295,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
             can_edit = has_studio_write_access(request.user, usage_key.course_key)
+            can_edit_visibility = not view_name == 'container_child_preview'
```

Comment:
I don't think that this rename really helps. It just leads one to wonder why children of containers can't have visibility settings.

What I meant is that the library view name should be renamed so that it is clear that it is specific to libraries. And the library code should be contained with libraries themselves, not put into views/item.py. Adding logic that is specific to libraries in this method is technical debt.